File: examples-sdk/go/306-rag-ai-event-tracking/main.go
// 306-rag-ai-event-tracking — Go SDK equivalent
// Compile: vil compile --from go --input 306-rag-ai-event-tracking/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("customer-support-rag", 3116)

	support := vil.NewService("support")
	support.Endpoint("POST", "/support/ask", "answer_question")
	s.Service(support)

	s.Compile()
}

File: examples-sdk/zig/306-rag-ai-event-tracking/main.zig
// 306-rag-ai-event-tracking — Zig SDK equivalent
// Compile: vil compile --from zig --input 306-rag-ai-event-tracking/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("customer-support-rag", 3116);
    var support = vil.Service.init("support");
    support.endpoint("POST", "/support/ask", "answer_question");
    server.service(&support);
    server.compile();
}
